api/pacman-api-asset/src/main/java/com/tmobile/pacman/api/asset/repository/CloudNotificationsRepositoryImpl.java

```diff
@@ -434,10 +434,10 @@ public List<Map<String, Object>> getCloudNotifications(String index, String type
 			String lte = null;
 
 			if ( startDate!= null) {
-				gte = "\"gte\": \"" + new SimpleDateFormat("yyyy-MM-dd").format(startDate) + "\"";
+				gte = "\"gte\": \"" + new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(startDate) + "\"";
 			}
 			if ( endDate != null) {
-				lte = "\"lte\": \"" + new SimpleDateFormat("yyyy-MM-dd").format(endDate) + "\"";
+				lte = "\"lte\": \"" + new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(endDate) + "\"";

```

Comment:
Here I would add a variable which will contain date format - seems they are the same.

Reply:
done